content/docs/conditional-rendering.md

```diff
@@ -70,9 +70,9 @@ function LogoutButton(props) {
 }
 ```
 
-In the example below, we will create a [stateful component](/docs/state-and-lifecycle.html#adding-local-state-to-a-class) called `LoginControl`.
+Pada contoh dibawah, kita akan membuat sebuah [komponen _stateful_](/docs/state-and-lifecycle.html#adding-local-state-to-a-class) `LoginControl`.
 
-It will render either `<LoginButton />` or `<LogoutButton />` depending on its current state. It will also render a `<Greeting />` from the previous example:
+Itu akan merender salah satu dari `<LoginButton />` atau `<LogoutButton />` berdasarkan _state_ saat ini. Itu juga akan merender `<Greeting />` dari contoh sebelumnya:
```

Comment:
Probably this, which refers to the example:

```suggestion
Contoh berikut akan merender salah satu dari `<LoginButton />` atau `<LogoutButton />` berdasarkan _state_ saat ini. Itu juga akan merender `<Greeting />` dari contoh sebelumnya:
```

Or this, which refers to this `LoginControl` component:

```suggestion
`LoginControl` akan merender salah satu dari `<LoginButton />` atau `<LogoutButton />` berdasarkan _state_ saat ini. Itu juga akan merender `<Greeting />` dari contoh sebelumnya:
```

Also, `merender` should be discussed further.